### ISSUE TITLE
Can't install using nvmw

### ISSUE THREAD

**Issue by simon-p-r** [hakobera/nvmw#37]
Getting this error when installing any version of both node and iojs

````SHELL 
A duplicate file name exists, or the file
cannot be found.
The system cannot find the path specified.
D:\
`````

**Comment by conradz** [comment]
I'm getting a similar error. I don't have any version of node or io.js installed globally. I'm using the latest nvmw commit (2c20d4c). Everytime I try to install I get this error:

```
A duplicate file name exists, or the file
cannot be found.
The system cannot find the path specified.
C:\Users\Conrad\.nvmw\latest
The process cannot access the file because it is being used by another process.
```

**Comment by anseki** [comment]
This error is solved by adding a line to `nvmw.bat`.

https://github.com/hakobera/nvmw/blob/master/nvmw.bat#L158
```
  set "CD_ORG=%CD%"
  cd "%NODE_HOME%"
```

Add a line `%~d0` to next of `set "CD_ORG=%CD%"`.

```
  set "CD_ORG=%CD%"
  %~d0
  cd "%NODE_HOME%"
```

Windows (DOS) has current-DIRECTORY and current-DRIVE. `cd` changes a current-DIRECTORY only.

**Comment by simon-p-r** [comment]
That has worked however when using the setting path command it doesn't persist to the next shell session however because I have node installed on my path I don't think I can set persistently anyway.

````SHELL
set "PATH=%HOMEDRIVE%%HOMEPATH%\.nvmw;%PATH%"

**Comment by hakobera** [comment]
@simon-p-r @conradz This issue is solved by #39
Thanks @anseki
